src/governance.cpp

```diff
@@ -947,8 +948,8 @@ bool CGovernanceManager::ProcessVote(CNode* pfrom, const CGovernanceVote& vote,
         std::ostringstream ostr;
         ostr << "CGovernanceManager::ProcessVote -- Old invalid vote "
                 << ", MN outpoint = " << vote.GetVinMasternode().prevout.ToStringShort()
-                << ", governance object hash = " << vote.GetParentHash().ToString() << "\n";
-        LogPrintf(ostr.str().c_str());
+                << ", governance object hash = " << vote.GetParentHash().ToString();
+        LogPrintf("%s\n", ostr.str());
```

Comment:
Because `ostr` is also used as a description string for an `exception` which should not have newline character (because, for example, it's used later in log output and this is causing empty line). `c_str()` is not needed for `LogPrintf` either.